plugins/docker/_docker

```diff
@@ -2652,7 +2653,7 @@ __docker_commands() {
     then
         local -a lines
         lines=(${(f)"$(_call_program commands docker 2>&1)"})
-        _docker_subcommands=(${${${(M)${lines[$((${lines[(i)*Commands:]} + 1)),-1]}:# *}## #}/\*# ##/:})
+        _docker_subcommands=(${${${(M)${lines[$((${lines[(i)*Commands:]} + 1)),-1]}:# *}## #}/ ##/:})
```

Comment:
This modification was to remove `*` from the subcommands:

### with
![image](https://user-images.githubusercontent.com/1441704/162021535-ac546869-0b99-4658-9fef-6c2285a2812d.png)

### without
![image](https://user-images.githubusercontent.com/1441704/162021526-f80e9fba-29ff-440d-b4e0-55a1a4f56069.png)

I fixed this in https://github.com/ohmyzsh/ohmyzsh/commit/c21ff38b8fda0584208ffd697dac0502e2327466 but forgot to send this upstream.

There's https://github.com/docker/cli/pull/2903 still to be merged, so the Zsh completion is probably not prioritary. Still, I'd like to wait until this patch is merged upstream so as to not introduce a regression.

Reply:
I've submitted the PR for this particular fix at https://github.com/docker/cli/pull/3540. You can subscribe to that to know when it's merged.

Reply:
It merged. So this is good to proceed?

Reply:
Hopefully! @mcornella what do you think?